.github/workflows/build.yml

```
@@ -73,7 +73,28 @@ jobs:
       build_qnap_arm: ${{ steps.decide.outputs.build_qnap_arm }}
       build_docker: ${{ steps.decide.outputs.build_docker }}
       build_linux_packages: ${{ steps.decide.outputs.build_linux_packages }}
+      version: ${{ steps.version.outputs.version }}
+      git_sha: ${{ steps.version.outputs.git_sha }}
     steps:
+      - name: Compute version
+        id: version
+        run: |
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2086:info:13:28: Double quote to prevent globbing and word splitting

(shellcheck)